deployment/NetworkTree.js

```diff
@@ -66,13 +66,15 @@ class NetworkTree {
      */
     removeNode(socket, socketID, room, root) {
         if(this.childNodes.length === 0) { return; }
-        this.childNodes.forEach((node) => {
-            if(node.socketID === socketID) {
-                this.notifyChildren(socket, node, room, root);
-            } else {
-                node.removeNode(socket, socketID, room, root);
+
```

Comment:
`this.childNodes.some((node) => {
            if(node.socketID === socketID) {
                this.notifyChildren(socket, node, room, root);
                return node.socketID === socketID;
            } else {
                node.removeNode(socket, socketID, room, root);
                return false
            }
            
        })`
can you switch this to something like this to remove the codacy issue.

Reply:
So, originally after the last PR when you switched it to forEach, the function stopped working. So I reverted it back. Let me see what I can do to fix codacy issues

Reply:
some function should work in this case